testsuite/tests/ui/conftest.py

```diff
@@ -28,21 +27,22 @@
 from testsuite.ui.views.devel.login import LoginDevelView
 from testsuite.ui.views.master.audience.tenant import TenantNewView
 from testsuite.ui.views.master.login import MasterLoginView
-from testsuite.ui.webdriver import SeleniumDriver
+from testsuite.ui.webdriver import Webdriver
 from testsuite.utils import blame
 
 
 @pytest.fixture(scope="session")
 def webdriver():
     """Creates instance of Web Driver with configuration"""
-    return SeleniumDriver(source=settings["fixtures"]["ui"]["browser"]["source"],
+    webdriver = Webdriver(source=settings["fixtures"]["ui"]["browser"]["source"],
```

Comment:
Is creating a variable necessary? The purpose is only to return an instance of webdriver.